website/server/libs/slack.js

```diff
@@ -0,0 +1,58 @@
+/* eslint-disable camelcase */
+import { IncomingWebhook } from '@slack/client';
+import logger from './logger';
+import { TAVERN_ID } from '../models/group';
+import nconf from 'nconf';
+
+const SLACK_FLAGGING_URL = nconf.get('SLACK:FLAGGING_URL');
+const SLACK_FLAGGING_FOOTER_LINK = nconf.get('SLACK:FLAGGING_FOOTER_LINK');
+const BASE_URL = nconf.get('BASE_URL');
+
+let flagSlack;
+
+try {
+  flagSlack = new IncomingWebhook(SLACK_FLAGGING_URL);
```

Comment:
This error catching is so the app doesn't crash if someone forgets to set the slack flagging url.

We could alternatively set the channel and bot info in the send call, rather than relying on the default values set for the slack endpoint.